doc/setup/installing_windows_cpp.adoc

```diff
@@ -0,0 +1,54 @@
+= Installing OSI for C++ on Windows
+
+**Prerequisites**
+
+* You have installed _cmake_ as administrator.
+* You have installed _protobuf_ as administrator.
+
+**Steps**
+
+. Open a terminal as administrator.
+. Clone the Open Simulation repository.
++
+[source]
+----
+git clone https://github.com/OpenSimulationInterface/open-simulation-interface.git
+----
++
+. Change into the repository directory.
++
+[source]
+----
+cd open-simulation-interface
+----
++
+. Create a new directory for the build.
++
+[source]
+----
+mkdir build
+----
++
+. Change into the new directory.
++
+[source]
+----
+cd build
+----
++
+. Run cmake.
+  To build a 64-bit target, add `Win64` to the generator name.
+  In this case, protobuf and protoc.exe must be in 64-bit mode too.
++
+[source]
+----
+cmake .. [-G <generator>] [-DCMAKE_INSTALL_PREFIX=<osi-install-directory>]
+----
++
+. Build and install OSI.
++
+[source]
+----
+cmake --build . [--config Release]
+cmake --build . --target install
+----
```

Comment:
Please review @pmai

Reply:
Looks good to me, any further tuning of installation guidance can be done when we improve installation/cmake stuff.